src/globals/type/_index.scss

```diff
@@ -72,42 +72,3 @@ $type-style-deprecation-map: (
   @warn deprecate($actual: 'type-scale-item($scale-item: #{$scale-item})', $expected: 'carbon--type-style($name: #{$name})');
   @include carbon--type-style($name: $name);
 }
-
```

Comment:
Could you also remove the `type-scale-item` helper method along with the deprecation maps and imports above?